packages/backend/src/build-disk-image.spec.ts

```diff
@@ -541,9 +541,7 @@ test('test building with a buildConfig JSON file that a temporary file for build
     buildConfig: buildConfig,
   } as BootcBuildInfo;
 
-  // Spy on fs.writeFileSync to make sure it is called
   vi.mock('node:fs');
```

Comment:
Shouldn't this be at the root of the file ?

Reply:
I'm assuming we should also be using vi.mocked() over vi.spyOn()? I was just trying to fix one problem at a time, but I did move this mock() to root in a new commit.